site/source/docs/compiling/WebAssembly.rst

```diff
@@ -63,10 +63,6 @@ upgrade from fastcomp to upstream:
     (``-flto``, ``-flto=full``, ``-flto=thin``, at both compile and link times).
     These flags will make the wasm backend behave more like fastcomp.
 
-  * With fastcomp LTO optimization passes will not be run by default;
-    for that you must pass ``--llvm-lto 1``.  With the llvm backend
-    LTO passes will be run on any object files that are in bitcode format.
```

Comment:
I think we should not remove this. As part of the docs on migrating from fastcomp to upstream, we should keep it around forever, or at least a very long time.